app/controllers/planning_applications_controller.rb

```diff
@@ -42,26 +41,38 @@ def assign
                                    else
                                      current_local_authority.users.find(params[:planning_application][:user_id])
                                    end
+      @planning_application.user.nil? ? audit("assigned") : audit("assigned", "", @planning_application.user.name)
```

Comment:
I think `audit("assigned", nil, @planning_application.user&.name)` is a neater way of doing this line